packages/waku/src/lib/builder/build.ts

```diff
@@ -135,6 +139,25 @@ const analyzeEntries = async (rootDir: string, config: ResolvedConfig) => {
       fname,
     ]),
   );
+  await buildVite({
+    plugins: [
+      rscAnalyzePlugin({ isClient: true, serverFileSet }),
+      rscManagedPlugin(config),
+    ],
+    ssr: {
+      target: 'webworker',
+      noExternal: /^(?!node:)/,
+    },
+    build: {
+      write: false,
+      ssr: true,
```

Comment:
on second look, i'm not sure if this should be "ssr". we will need to revisit this double build anyway.